src/constants/entities.asm

```diff
@@ -272,4 +272,4 @@ ENTITY_PHOTOGRAPHER                  equ $FA
 ; unused                             equ $FC
 ; unused                             equ $FD
 ; unused                             equ $FE
-; unused                             equ $FF
+ENTITY_EMPTY                         equ $FF
```

Comment:
Nitpick: by analogy with other constants, I would rather name this constant `ENTITY_NONE`.

Reply:
ok will be part of the next pull request